js/signup.js

```diff
@@ -0,0 +1,74 @@
+const base_api = "https://floesia-api.herokuapp.com";
+
+window.onload = () => {
+    const signupBtn = document.querySelector("#signupBtn");
+    signupBtn.addEventListener("click", () => {
+        clearAlertBox();
+        signup();
+    })
+}
+
+function signup() {
+    const data = getFormData();
+    fetch( `${base_api}/signup`, genPostData(data) )
+        .then( data => {
+
+            console.log(data.status);
+
+            if (data.status === 404) {
+                redAlert('Incorrect nickname, email or password.')
+            } else {
+                data
+                    .json()
+                    .then(data => {
+                        storeToken(data.token);
+                        location.href = '/';
```

Comment:
You can redirect the client to the login page here, with ```location.href = '/login';```.

Reply:
done.
thanks! =]